subsys/bluetooth/shell/gatt.c

```diff
@@ -379,8 +380,24 @@ int cmd_gatt_write_without_rsp(int argc, char *argv[])
 		}
 	}
 
-	err = bt_gatt_write_without_response(default_conn, handle,
-					     gatt_write_buf, len, sign);
+	repeat = 0;
+
+	if (argc > 4) {
+		repeat = strtoul(argv[4], NULL, 16);
+	}
+
+	if (!repeat) {
+		repeat = 1;
```

Comment:
Wouldn't it be simpler just to initialize repeat to 1 instead of 0 to start with?